### PR TITLE
Fut 27

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

File: Future/inst/IG.txt [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+P�atki j�czmienne	40			
+P�atki pszenne	50			
+P�atki �ytnie	40			
+Porzeczki bia�e	20			
+Poziomki	35			
+Rze�ucha	10			
+Rzodkiewka	15			
+Sok ananasowy	50			
+Sok z czarnej porzeczki	25			
+
```
Comment (darsoo) [inline Future/inst/IG.txt:231]: remove empty rows

File: Future/inst/digestible_carbo.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
+nazwa	weglowodany	blonnik	przyswajalne_weglo
```
Comment (darsoo) [inline Future/inst/digestible_carbo.txt:1]: use english names

File: Future/inst/glycemic_table.txt [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+"Majonez";60
+"Mi�d pszczeli";60
+"Mleko 3,2% t�uszczu";60
+"Ry� d�ugoziarnisty";60
+"Chleb pszenny razowy";65
+"Kukurydza, konserwa";65
+"Kasza kuskus";65
+"Rodzynki";65
+"Ziemniaki";65
+"Bu�ka grahamka";70
```
Comment (darsoo) [inline Future/inst/glycemic_table.txt:146]: fix unexpected sign

File: Future/R/glycemic_index.R [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                        msg = "elements of list_of_products must be character")
+  assertthat::assert_that(is.list(weight_of_products),
+                        msg = "weight_of_products must be list")
+  assertthat::assert_that(all(sapply(weight_of_products, class) == "numeric"),
+                        msg = "elements of weight_of_products must be numeric")
+  assertthat::assert_that(length(list_of_products) == length(weight_of_products),
+                        msg = "list_of_products and weight_of_products must be the same length")
+
+ dataframe_IG <- data.frame()
+  for (i in 1:length(list_of_products)) {
```
Comment (darsoo) [inline Future/R/glycemic_index.R:68]: use seq_len

File: Future/R/glycemic_index.R [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+#' list_of_products <- list("Banan", "Truskawki")
+#' weight_of_products <- list(100, 100)
+#' glycemic_index_of_meal(list_of_products, weight_of_products)
+#'
+#' @export
+glycemic_index_of_meal <- function(list_of_products, weight_of_products){
+
+  assertthat::assert_that(is.list(list_of_products),
+                          msg = "list_of_products must be list")
+  assertthat::assert_that(all(sapply(list_of_products, class) == "character"),
```
Comment (darsoo) [inline Future/R/glycemic_index.R:58]: vapply is safer

File: Future/R/glycemic_index.R [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  sum_digestible_carbo <- colSums(dataframe_IG[4])
+  percent_digestible_carbo <- round(dataframe_IG$digestible_carbo / sum_digestible_carbo, 4)
+  dataframe_IG <- cbind(dataframe_IG, percent_digestible_carbo)
+
+  IG_of_product <- round(dataframe_IG$percent_digestible_carbo * dataframe_IG$IG, 2)
+  dataframe_IG <- cbind(dataframe_IG, IG_of_product)
+
+  sum_IG_of_meal <- sum(dataframe_IG$IG_of_product)
+
+  sum_IG_of_meal}
```
Comment (darsoo) [inline Future/R/glycemic_index.R:82]: move bracket to new line

**Comment by darsoo** [comment]
minor changes, and good to go.

File: Future/inst/IG.txt (hunk 1 above)
Comment (karollayna) [reply Future/inst/IG.txt:231]: done

File: Future/inst/digestible_carbo.txt (hunk 2 above)
Comment (karollayna) [reply Future/inst/digestible_carbo.txt:1]: done

File: Future/inst/glycemic_table.txt (hunk 3 above)
Comment (karollayna) [reply Future/inst/glycemic_table.txt:146]: done

File: Future/R/glycemic_index.R (hunk 4 above)
Comment (karollayna) [reply Future/R/glycemic_index.R:68]: done

File: Future/R/glycemic_index.R (hunk 6 above)
Comment (karollayna) [reply Future/R/glycemic_index.R:82]: done

**Review by darsoo — APPROVED** [review]

**Pull request merged**